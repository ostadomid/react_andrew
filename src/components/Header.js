import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <ul className="menu-bar">
        <li className="menu-item"> <NavLink exact={true} activeClassName="red" to="/">Home</NavLink> </li>    
        <li className="menu-item"> <NavLink exact={true} activeClassName="red" to="/add">Add</NavLink> </li>    
        <li className="menu-item"> <NavLink exact={true} activeClassName="red" to="/edit">Edit</NavLink> </li>    
    </ul>    
);

export default Header;