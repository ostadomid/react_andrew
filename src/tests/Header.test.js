import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../components/Header';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';



test('Rendering Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.find('li').length).toBe(3);
    expect(toJson(wrapper)).toMatchSnapshot();
});
