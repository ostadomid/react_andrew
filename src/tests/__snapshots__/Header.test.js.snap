// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering Header 1`] = `
<ul
  className="menu-bar"
>
  <li
    className="menu-item"
  >
     
    <NavLink
      activeClassName="red"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      Home
    </NavLink>
     
  </li>
  <li
    className="menu-item"
  >
     
    <NavLink
      activeClassName="red"
      ariaCurrent="true"
      exact={true}
      to="/add"
    >
      Add
    </NavLink>
     
  </li>
  <li
    className="menu-item"
  >
     
    <NavLink
      activeClassName="red"
      ariaCurrent="true"
      exact={true}
      to="/edit"
    >
      Edit
    </NavLink>
     
  </li>
</ul>
`;
